### PR TITLE
Added the blacklisting of a peer

### PR DESCRIPTION
**Pull request by Millymanz**
Added the blacklisting of a peer

### CODE REVIEW

File: src/Rpc.proto [hunk 1]
```diff
@@ -184,6 +184,18 @@ message GetPeerReputationResponse {
    int32 reputation = 1;
 }
 
+message GetPeerBlackListRequest {
+    bytes publicKey = 1;
+    bytes ip = 2;
+	bool blacklist = 3;
```
Comment (franssl) [inline src/Rpc.proto]:
```suggestion
    bool blacklist = 3;
```

File: src/Rpc.proto [hunk 2]
```diff
@@ -184,6 +184,18 @@ message GetPeerReputationResponse {
    int32 reputation = 1;
 }
 
+message GetPeerBlackListRequest {
```
Comment (monsieurleberre) [inline src/Rpc.proto]: As discussed, this name is a bit more accurate if you want to change the peer's status
```suggestion
message UpdatePeerBlackListRequest {
```

File: src/Rpc.proto [hunk 3]
```diff
@@ -184,6 +184,18 @@ message GetPeerReputationResponse {
...
+    bytes publicKey = 1;
+    bytes ip = 2;
+	bool blacklist = 3;
+}
+
+message GetPeerBlackListResponse{
```
Comment (monsieurleberre) [inline src/Rpc.proto]:
```suggestion
message UpdatePeerBlackListResponse{
```

**Review by monsieurleberre — CHANGES_REQUESTED** [review]
see suggestions

**Review by monsieurleberre — DISMISSED** [review]
OK for me

**Review by monsieurleberre — APPROVED** [review]

**Review by monsieurleberre — APPROVED** [review]

**Review by nshCore — APPROVED** [review]

**Pull request merged**